ci/infra/testrunner/platforms/terraform.py

```diff
@@ -3,88 +3,26 @@
 import os
 import subprocess
 
-from timeout_decorator import timeout
-
 from utils import (Format, step, Utils)
 
 
 class Terraform:
-    def __init__(self, conf, platform):
+    def __init__(self, conf, platform, env_setup):
         self.conf = conf
+        self.env_setup = env_setup if env_setup else ""
```

Comment:
I don't see the advantage of using a variable instead of a function for setting the environment. A function is more flexible because it allows other setups.